pkg/github/generate.go

```diff
@@ -0,0 +1,70 @@
+package github
+
+import (
+	"context"
+	"encoding/base64"
+	"github.com/bradleyfalzon/ghinstallation/v2"
+	"log"
```

Comment:
you probably wanted `log "github.com/sirupsen/logrus"` for consistency

Reply:
yes